lib/llvm-backend/cpp/object_loader.cpp

```diff
@@ -1,10 +1,60 @@
 #include "object_loader.hh"
 #include <iostream>
 #include <memory>
+#include <setjmp.h>
 
 extern "C" void __register_frame(uint8_t *);
 extern "C" void __deregister_frame(uint8_t *);
 
+void unwinding_setjmp(jmp_buf stack_out, void (*func)(void *), void *userdata) {
```

Comment:
```
#ifdef __GNUC__
__attribute__((returns_twice))
#endif
void unwinding_setjmp(jmp_buf stack_out, void (*func)(void *), void *userdata) {
```

Reply:
What `unwinding_setjmp` does is to call the callback `func` and catch `longjmp`'s backward.  It only returns once and I think `returns_twice` is not needed.

Reply:
Oh yes, you're correct.